src/cascadia/TerminalSettingsModel/PowershellCoreProfileGenerator.cpp

```diff
@@ -294,6 +295,16 @@ std::wstring_view PowershellCoreProfileGenerator::GetNamespace() const noexcept
     return PowershellCoreGeneratorNamespace;
 }
 
+std::wstring_view PowershellCoreProfileGenerator::GetDisplayName() const noexcept
+{
+    return RS_(L"PowershellCoreProfileGeneratorDisplayName");
+}
+
+std::wstring_view PowershellCoreProfileGenerator::GetIcon() const noexcept
+{
+    return POWERSHELL_ICON;

```

Comment:
This icon doesn't look that great since I'm resizing it to make it bigger:
![{960B37ED-8DD9-4979-938B-32E3232FD560}](https://github.com/user-attachments/assets/515209ec-aad8-4033-9106-ca8f82a9cc04)

Is there a better resource I can use here?